### PR TITLE
Curvebs stop volume snapshot

### PR DESCRIPTION
**Pull request by ZackSoul**
### What problem does this PR solve?

Issue Number: #2579 

Problem Summary:

### What is changed and how it works?
add curve bs stop volume snapshot

What's Changed:

How it Works:
curve bs stop volume volumeSnapshot

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
pls fix dco

**Review by Cyber-SiKu — COMMENTED** [review]
add in readme

**Comment by ZackSoul** [comment]
Keep reviewing？Please.

File: tools-v2/pkg/config/bs.go [hunk 1]
```diff
@@ -630,6 +633,10 @@ func AddBsTaskIDOptionFlag(cmd *cobra.Command) {
 	AddBsStringOptionFlag(cmd, CURVEBS_TASKID, "task id")
 }
 
+func AddBsSnapshotSeqIDOptionFlag(cmd *cobra.Command) {
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/config/bs.go:636]: is optional or required?

File: tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/volumeSnapshot.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+func (sCmd *StopCmd) RunCommand(cmd *cobra.Command, args []string) error {
+	s := newStopSnapShot(sCmd.snapshotAddrs, sCmd.timeout, sCmd.user, sCmd.fileName, sCmd.uuid)
+	records := s.queryStopBy()
+	for _, item := range records {
+		err := s.stopSnapShot(item.UUID, item.User, item.File)
+		if err == nil {
+			item.Result = cobrautil.ROW_VALUE_SUCCESS
+		} else {
+			item.Result = cobrautil.ROW_VALUE_FAILED
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/volumeSnapshot.go:53]: Refer to the error output of the original tool, please enrich it.

File: tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/volumeSnapshot.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+		}
+		sCmd.TableNew.Append([]string{item.Result})
+	}
+	sCmd.Result = records
+	sCmd.Error = cmderror.Success()
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/volumeSnapshot.go:58]: Is it a success if a deletion error occurs?

File: tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+)
+
+const (
+	version = "0.0.6"
+	limit   = "100"
+	offset  = "0"
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go:16]: capital

File: tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+	var resp struct {
+		Code       string
+		Message    string
+		RequestId  string
+		TotalCount int
+		SnapShots  []*Record
+	}
+	err := s.query(params, &resp)
+	if err != nil || resp.Code != "0" {
+		fmt.Printf("get snapshot list fail, error=%s\n", err)
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go:97]: Try not to output errors in the middle, but return an error, cmdError, etc.
You can refer to the implementation of other commands

**Review by Cyber-SiKu — CHANGES_REQUESTED** [review]

File: tools-v2/pkg/config/bs.go (hunk 1 above)
Comment (ZackSoul) [reply tools-v2/pkg/config/bs.go]: optional

**Review by Cyber-SiKu — APPROVED** [review]

File: tools-v2/pkg/config/bs.go [hunk 6]
```diff
@@ -634,6 +637,10 @@ func AddBsFailedOptionFlag(cmd *cobra.Command) {
 	AddBsBoolOptionFlag(cmd, CURVEBS_FAILED, "failed")
 }
 
+func AddBsSnapshotSeqIDOptionFlag(cmd *cobra.Command) {
+	AddBsStringOptionFlag(cmd, CURVEBS_SNAPSHOTSEQID, "snapshot seqId")
+}
+
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/config/bs.go:643]: This value should be required, right?

**Comment by ZackSoul** [comment]
cicheck

**Comment by ZackSoul** [comment]
cicheck

**Comment by ZackSoul** [comment]
cicheck

**Comment by ZackSoul** [comment]
cicheck

**Comment by ZackSoul** [comment]
cicheck

**Comment by ZackSoul** [comment]
cicheck

**Comment by ZackSoul** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
fix conflicts & dco

**Comment by ZackSoul** [comment]
cicheck

**Pull request closed without merging**